runtime/doc/nvim_intro.txt

```diff
@@ -6,19 +6,20 @@
 
 Introduction to Nvim					*nvim* *nvim-intro*
 
-This is an introduction to Vim users that are just getting started with Nvim.
+This is an introduction for Vim users who are just getting started with Nvim.
 It is not meant for Vim beginners. For a basic introduction to Vim,
 see |help.txt|.
 
 For now, it is just an index with the most relevant topics/features that
 differentiate Nvim from Vim:
 
-1. Msgpack-RPC			|msgpack-rpc|
-2. Job control			|job-control|
-3. Python plugins		|nvim-python|
-4. Clipboard integration	|nvim-clipboard|
-5. Remote plugins		|remote-plugin|
-6. Provider infrastructure	|nvim-provider|
+1. Differences from Vim		|vim-differences|
```

Comment:
It looks fine in nvim, which reads `ts=8` from the modeline. GitHub looks like it uses 4.